Tests/Tests/STPEphemeralKeyManagerTest.m

```diff
@@ -32,12 +31,16 @@ - (void)setUp {
     self.apiVersion = @"2015-03-03";
 }
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated"
```

Comment:
what are we using here that's deprecated?

Reply:
@jack-stripe ?

Reply:
My mistake, I added this accidentally in my initial testing, feel free to remove these ignores

Reply:
👍